.github/workflows/cd.yml

```diff
@@ -0,0 +1,30 @@
+name: CD
+
+on: [release]
+
+jobs:
+  publish-chart:
+    runs-on: ubuntu-latest
+    needs: test-chart
+    if: "startsWith(github.ref, 'refs/tags/') || github.ref == 'refs/heads/master'"
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    - name: set up environment variables
+      run: |
+        echo "GIT_USER=Renku Bot" >> $GITHUB_ENV
+        echo "GIT_EMAIL=renku@datascience.ch" >> $GITHUB_ENV
+    - name: Push chart and images
+      uses: SwissDataScienceCenter/renku/actions/publish-chart@master
+      env:
+        GITHUB_TOKEN: ${{ secrets.DEPLOY_TOKEN }}
+        DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
+        DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
+    - name: Wait for chart to get published
+      run: sleep 120
+    - name: Update component version
+      uses: SwissDataScienceCenter/renku/actions/update-component-version@master
```

Comment:
The PR that is triggered by this action will need to be changed back to merge to `master`

Reply:
Yep - the corresponding issue is here: https://github.com/swissdatasciencecenter/renku/issues/1914